### ISSUE TITLE
ACACA gene ENSG00000132142 is deprecated and possibly replaced by ENSG00000278540

### ISSUE THREAD

**Issue by egonw** [wikipathways/reactome-curation#1]
* WikiPathways: https://www.wikipathways.org/index.php/Pathway:WP2706 (Reactome 74)
* Reactome: https://reactome.org/PathwayBrowser/#DIAGRAM=2426168

**Comment by egonw** [comment]
cc @Chris-Evelo, you asked me about us reporting curation events with Reactome.

**Comment by egonw** [comment]
The issue still exists in Reactome. Ping @hawrobin

**Comment by Chris-Evelo** [comment]
Why "possible outdated"? don't we know for sure?

**Comment by egonw** [comment]
Yeah, they are outdated, but there is not always a replacement for what the pathway is telling. I can imagine there sometimes a reason why the "outdated" identifier should be kept. But that needs individual inspection.
